packages/ignite-cli/src/commands/plugin.js

```diff
@@ -20,8 +20,14 @@ const walkthrough = async (context) => {
   // Okay, we'll ask one by one, fine
   return await context.prompt.ask([
     {
+      name: 'boilerplate',
+      message: 'Is this an app boilerplate plugin?',
```

Comment:
You're probably right. I like having a question, can you think of another wording that might make sense?